src/DataService/DataServiceManager.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Lullabot\Mpx\DataService;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+
+class DataServiceManager
+{
+    /**
+     * @var \Lullabot\Mpx\DataService\DataServiceDiscovery
+     */
+    private $discovery;
+
+    public function __construct(DataServiceDiscovery $discovery)
+    {
+        $this->discovery = $discovery;
+    }
+
+    /**
+     * Register our annotations, relative to this file.
+     *
+     * @return static
+     */
+    public static function basicDiscovery()
+    {
+        // @todo Check Drupal core for other tags to ignore?
+        AnnotationReader::addGlobalIgnoredName('class');
+        AnnotationRegistry::registerFile(__DIR__.'/Annotation/DataService.php');
```

Comment:
This line is critical, otherwise `Access` is loaded first, which uses the DataService annotation, and you end up with a class not found fatal.